splot/plot.py

```diff
@@ -4,31 +4,37 @@
 
 __author__ = "Marynia Kolak <marynia.kolak@gmail.com>"
 
-import pandas as pd
-import numpy as np
 import pysal as ps
+import esda
 import matplotlib.pyplot as plt
+from matplotlib import colors
 
 
 __all__ = ['mplot']
 
 
-def mplot(m, xlabel='', ylabel='', title='', custom=(7,7)):
+def mplot(m, xlabel='', ylabel='', title='', figsize=(7,7), p=None, ax=None,
```

Comment:
I don't know to what extent you are bound to historical names, but for clarity and for consistency with the other functions, I would personally rather call this something like `plot_moran`

Reply:
Also, should this be moved to `splot.mpl` ? (or at least also exposed there, so people can import it from there)

Reply:
Good point, I also added this to the API discussion: https://github.com/pysal/splot/issues/9